invokeai/app/api/routers/workflows.py

```diff
@@ -105,6 +105,7 @@ async def list_workflows(
     categories: Optional[list[WorkflowCategory]] = Query(default=None, description="The categories of workflow to get"),
     tags: Optional[list[str]] = Query(default=None, description="The tags of workflow to get"),
     query: Optional[str] = Query(default=None, description="The text to query by (matches name and description)"),
+    is_recent: Optional[bool] = Query(default=None, description="Whether to include/exclude recent workflows"),
```

Comment:
technically this should be `has_been_opened` or something - wdyt @psychedelicious